pyipn/detector.py

```diff
@@ -21,15 +21,18 @@ def __init__(self, location, pointing, effective_area, name):
         :rtype: 
 
         """
-
+     
         self._effective_area = effective_area
         self._pointing = pointing
         self._location = location
         self._background_slope = 0.0
-        self._background_norm = 500.0
-
         self._name = name
+        if re.search('GALILEO', self._name) is not None:
+            self._background_norm = 3840.0 * self._effective_area.total_area / 360.0
+        else:
+            self._background_norm = 500.0
 
```

Comment:
It is either better to inherit this or do the logic outside the class as an input because making this even more general will require losts of switches and make the OO framework fragile.